modules/light-clients/06-solomachine/types/update.go

```diff
@@ -1,11 +1,17 @@
 package types
 
 import (
+	"encoding/hex"
+
+	"github.com/armon/go-metrics"
 	"github.com/cosmos/cosmos-sdk/codec"
+	"github.com/cosmos/cosmos-sdk/telemetry"
 	sdk "github.com/cosmos/cosmos-sdk/types"
 	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
 
+	"github.com/cosmos/ibc-go/v3/modules/core/02-client/types"
```

Comment:
alias should be clienttypes. Any `types` package not from this submodule should have an alias to distinguish which types package is being used. (ie clienttypes, connectiontypes, channeltypes, etc)